benchmark/src/test/java/com/datatorrent/benchmark/memsql/MemsqlInputBenchmarkTest.java

```diff
@@ -77,8 +77,8 @@ public void testMethod() throws SQLException, IOException
     attributeMap.put(OperatorContext.PROCESSING_MODE, ProcessingMode.AT_LEAST_ONCE);
     attributeMap.put(OperatorContext.ACTIVATION_WINDOW_ID, -1L);
     attributeMap.put(DAG.APPLICATION_ID, APP_ID);
-    OperatorContextTestHelper.TestIdOperatorContext context =
-        new OperatorContextTestHelper.TestIdOperatorContext(OPERATOR_ID, attributeMap);
+    OperatorContextTestHelper.MockOperatorContext context =
```

Comment:
You would need to ask that question to the original author. You see it's not introduced by this PR. As I said, it should be a separate discussion, it is not the scope of this PR (regardless whether we agree or not).